scripts/dasharo-deploy

```diff
@@ -810,6 +810,10 @@ usage() {
   echo "  $0 restore  - Restore from a previously saved backup"
 }
 
+if ! can_install_dasharo; then
```

Comment:
This has to be used with care. The fused bit in HFSTS6 does not apply to all platforms. Dell Optiplex for example does not have the fused bit. Same with PC Engines platforms. It only applies to a subset of Intel platforms. So basically this will break deployments on many platforms that we support.

Reply:
Same as [here](https://github.com/Dasharo/dts-scripts/pull/15#discussion_r1671720131)

Reply:
To not break deployments and updates we should only check for Boot Guard on platforms that we want to check for it.

Reply:
Now it works like this: First the script checks if the CPU vendor is Intel. If not, the other checks are skipped. If it's an Intel platform, but `rdmsr` command fails, then we assume that Boot Guard is not supported and the update runs. d407c14bfb8efaccb8475b979738f1e46d6bffa6